src/test/java/Interactors/GameCreationTest.java

```diff
@@ -0,0 +1,150 @@
+        Property medAvenue = new Property ("Mediterranean Avenue","Brown",60,50,medAvenueRent,expectedPlayer01,30,0,true);
+        int[] illAvenueRent = new int[]{20,100,300,750,925,1100};
+        Property illAvenue = new Property ("Illinois Avenue","Red",240,150,illAvenueRent,expectedPlayer01,120,0,false);
+        Player expectedPlayer02 = new Player("Player02", 200, true, 0, 10);
+        int[] bWalkRent = new int[]{50,200,600,1400,1700,2000};
+        Property bWalk = new Property ("Boardwalk","Dark Blue",400,200,bWalkRent,expectedPlayer02,200,0,false);
+        expectedPlayer01.addProperty(medAvenue);
+        expectedPlayer01.addProperty(illAvenue);
+        expectedPlayer02.addProperty(bWalk);
+
+        expectedPlayers.add(expectedPlayer01);
+        expectedPlayers.add(expectedPlayer02);
+
+        for (int i = 0; i < returnedPlayers.size(); i++){
+            assert expectedPlayers.get(i).getName().equals(returnedPlayers.get(i).getName());
+            assert expectedPlayers.get(i).getMoney() == returnedPlayers.get(i).getMoney();
+            assert expectedPlayers.get(i).getJailCards() == returnedPlayers.get(i).getJailCards();
+            assert expectedPlayers.get(i).getPosition() == returnedPlayers.get(i).getPosition();
+            assert expectedPlayers.get(i).isInJail() == returnedPlayers.get(i).isInJail();
+
+            ArrayList<Property> expectedProperties = expectedPlayers.get(i).getProperties();
+            ArrayList<Property> returnedProperties = returnedPlayers.get(i).getProperties();
+            for (int k = 0; k < returnedProperties.size(); k++){
+                // assert property and property owner are the same for expected and returned properties
+                assert expectedProperties.get(k).getName().equals(returnedProperties.get(k).getName());
+                assert expectedProperties.get(k).getOwner().getName().equals(returnedProperties.get(k).getOwner().getName());
+            }
+        }
+
+        //create expected cell values
+        ArrayList<Cell> cellProperties = gameCreator.parsePropertyData(properties);
+        ArrayList<Cell> propertiesSoFar = new ArrayList<>();
+        propertiesSoFar.add(medAvenue);
+        propertiesSoFar.add(illAvenue);
+        propertiesSoFar.add(bWalk);
+        List<Cell> expectedCells = gameCreator.createCells(propertiesSoFar, cellProperties);
+
+        for (int j = 0; j < returnedCells.size(); j++) {
+            assert returnedCells.get(j).getType().equals(expectedCells.get(j).getType());
+            if (returnedCells.get(j).getType().equals("Property")){
+                Property currCell = (Property) returnedCells.get(j);
+                if (currCell.getOwner() != null) {
+                    Property expectedCell = (Property) expectedCells.get(j);
+                    assert currCell.getOwner().getName().equals(expectedCell.getOwner().getName());
+                }
+            }
+        }
+
+    }
+}
```

Comment:
This function is really long

Reply:
Renamed the test functions so that there is no confusion in the future